src/Graph.cpp

```diff
@@ -85,10 +86,10 @@ void Graph::serializeGraph(Graph& G, string outputName, bool typedNodes, bool lo
             close(fd);
             return;
         }
-        string outName = "networks/" + outputName + "/autogenerated/" + outputName + (typedNodes ? "_TypedNodes" : "_UntypedNodes")
+        string outName = "networks/" + outputName + "/autogenerated/" + outputName + (bipartite ? "_bipartite" : "_Unbipartite")
```

Comment:
Since we will be adding true typed nodes soon, "Unbipartite" probably isn't the correct name here. Just leave it "UntypedNodes".